### ISSUE TITLE
Custom CSS doesn't copy from assets to build directory

### ISSUE THREAD

**Issue by gleicher** [Vimux/Mainroad#162]
Because of the way the permalinks are loaded with customCSS files, the assets are not copies over (even though the links are placed into the html).

The fix is to replace line 21 of baseof.html with the two lines:
    {{ $style := resources.Get . }}
    <link rel="stylesheet" href="{{ $style.RelPermalink }}" integrity="{{ $style.Data.Integrity }}" crossorigin="anonymous" media="screen">

These lines are taken from: https://discourse.gohugo.io/t/where-do-i-edit-a-themes-css/18551/7

The explanation for why it works is on: 
https://gohugo.io/getting-started/directory-structure/

which says:
Only the files whose .Permalink or .RelPermalink are used will be published to the public directory.

Sorry that I can't do a pull request (I don't have the theme as a separate repo to clone)

**Comment by contributorpw** [comment]
I'm note sure that it works like in the topic.

As you can see the theme from there uses proc of SASS.

For the Mainroad the solution from [here](https://discourse.gohugo.io/t/static-css-changes-no-updating-browser-cache-with-hugo-serve/16169/2?u=oshliaer) is more suitable.

As for me I'm removing public and resources as often as possible.

**Comment by gleicher** [comment]
I don't think it is a caching issue - I tried turning off the cache (as the page you point to suggests), but it doesn't help. It could be you are accessing the CSS resources from somewhere else that makes things work for you.

You are correct that my solution doesn't account for SASS (neither does the default baseof). It also didn't do template expansion. So, here is a more correct solution (OK, you can see I am a go template newbie since I don't know how to make $. work, so I used the temp variable)

		{{ $dot := . -}}
		{{ range .Site.Params.customCSS -}}
		{{ $style := resources.Get . | resources.ExecuteAsTemplate . $dot | resources.ToCSS }}
	  <@link rel="stylesheet" href="{{ $style.RelPermalink }}" crossorigin="anonymous" @>
		{{- end -}}

(note: I put in @ to avoid this being interpreted as HTML)

This may run all CSS files through the SASS processor - depending on how smart ToCSS is. This shouldn't be a problem (CSS is valid SASS).

**Comment by Vimux** [comment]
> Custom CSS doesn't copy from assets to build directory

It's not provided for. There is no mention in repository that custom CSS (or JS) should work with assets files.

Your proposal is not backward compatible and handles assets files only.
